cmake/projects/RapidJSON/hunter.cmake

```diff
@@ -9,6 +9,17 @@ include(hunter_cmake_args)
 include(hunter_download)
 include(hunter_pick_scheme)
 
+hunter_add_version(
+    PACKAGE_NAME
+    RapidJSON
+    VERSION
+    "1.1.0-66eb606-p0"
+    URL
+    "https://github.com/hunter-packages/rapidjson/archive/v1.1.0-66eb606-p0.tar.gz"
+    SHA1
+    947c0196612c9be6c39c496f84b03234495af5b1
```

Comment:
Should be
```
SHA1
6889242b198cbff7cd8610f214b4218436a4a7a9
```
intead of
```
SHA1
947c0196612c9be6c39c496f84b03234495af5b1
```

Reply:
My mistake - I updated the URL but forgot the SHA. Will do now

Reply:
```
$ openssl sha1 v1.1.0-66eb606-p0.tar.gz 
SHA1(v1.1.0-66eb606-p0.tar.gz)= 6889242b198cbff7cd8610f214b4218436a4a7a9
```